pkg/jmespath/jmespath.go

```diff
@@ -0,0 +1,203 @@
+package jmespath
```

Comment:
Maybe this file is better named `pkg/jmespath/expression.go`, as that's the primary type that this file declares. At least, that would be more consistent with the rest of the code base.